### PR TITLE
Add max 'Max characters length' to Admin-UI

### PR DESCRIPTION
**Pull request by autogun**

### CODE REVIEW

File: web-src/src/admin/components/scripts-config/ParameterConfigForm.vue [hunk 1]
```diff
@@ -67,6 +67,9 @@
                        title="Allowed file extensions"
                        v-model="fileExtensions"/>
         </div>
+        <div class="row" v-if="selectedType === 'text'">
```
Comment (bugy) [inline web-src/src/admin/components/scripts-config/ParameterConfigForm.vue:70]:
```suggestion
        <div class="row" v-if="selectedType === 'text' || selectedType === undefined">
```

File: web-src/src/admin/components/scripts-config/ParameterConfigForm.vue [hunk 2]
```diff
@@ -67,6 +67,9 @@
...
+            <Textfield :config="maxLengthField" @error="handleError(maxLengthField, $event)" class="col s5" v-model="max_length"/>
```
Comment (bugy) [inline web-src/src/admin/components/scripts-config/ParameterConfigForm.vue:71]:
```suggestion
            <Textfield :config="maxLengthField" @error="handleError(maxLengthField, $event)" class="col s4" v-model="max_length"/>
```

**Review by bugy — CHANGES_REQUESTED** [review]
Hi, I have a couple of improvements suggestion.
Sorry for taking so long to check, but my checked out code was in a broken state (for the styling feature)

**Review by autogun — COMMENTED** [review]
Proposed changes

**Comment by bugy** [comment]
Cool, thanks a lot!

**Review by bugy — APPROVED** [review]

**Pull request merged**